Swiftfin/Views/ItemView/ScrollViews/iPadOSCinematicScrollView.swift

```diff
@@ -105,83 +98,92 @@ extension ItemView.iPadOSCinematicScrollView {
         var viewModel: ItemViewModel
 
         var body: some View {
-            HStack(alignment: .bottom) {
-
-                VStack(alignment: .leading, spacing: 20) {
-
-                    ImageView(viewModel.item.imageSource(
-                        .logo,
-                        maxWidth: UIScreen.main.bounds.width * 0.4,
-                        maxHeight: 130
-                    ))
-                    .placeholder { _ in
-                        EmptyView()
-                    }
-                    .failure {
-                        Text(viewModel.item.displayTitle)
-                            .font(.largeTitle)
-                            .fontWeight(.semibold)
-                            .lineLimit(2)
-                            .multilineTextAlignment(.leading)
-                            .foregroundColor(.white)
-                    }
-                    .aspectRatio(contentMode: .fit)
-                    .frame(maxWidth: UIScreen.main.bounds.width * 0.4, maxHeight: 130, alignment: .bottomLeading)
-
-                    ItemView.OverviewView(item: viewModel.item)
-                        .overviewLineLimit(3)
-                        .taglineLineLimit(2)
-                        .foregroundColor(.white)
-
-                    if viewModel.item.type != .person {
-                        HStack(spacing: 30) {
-                            DotHStack {
-                                if let firstGenre = viewModel.item.genres?.first {
-                                    Text(firstGenre)
-                                }
-
-                                if let premiereYear = viewModel.item.premiereDateYear {
-                                    Text(premiereYear)
-                                }
-
-                                if let playButtonitem = viewModel.playButtonItem, let runtime = playButtonitem.runTimeLabel {
-                                    Text(runtime)
+            GeometryReader { geometry in
+                HStack(alignment: .bottom) {
+
+                    VStack(alignment: .leading, spacing: 20) {
+
+                        ImageView(viewModel.item.imageSource(
+                            .logo,
+                            maxWidth: geometry.size.width * 0.4,
```

Comment:
For the image size, I would like us to choose a static size so that only a single image is cached and used in the event of resizing as the image cache key is currently determined by the max width. We can choose something that "looks good enough".